assets/js/project.js

```diff
@@ -29,13 +29,37 @@ const $appLanguage = $('#app-language')
 let editor = null
 
 if ($project.data('my-program')) {
-  new MDCTextField(document.querySelector('#edit-mdc-text-field'))
   new MDCTextField(document.querySelector('.comment-message'))
 
-  editor = new ProjectTextEditor(
+  const nameEditorTextField = new ProjectEditorTextField(
     $projectDescriptionCredits,
-    $projectDescriptionCredits.data('trans-default'),
-    $projectDescriptionCredits.data('project-id')
+    $projectDescriptionCredits.data('project-id'),
+    'name',
+    true
+  )
+
+  const descriptionEditorTextField = new ProjectEditorTextField(
+    $projectDescriptionCredits,
+    $projectDescriptionCredits.data('project-id'),
+    'description',
+    $projectDescriptionCredits.data('has-description')
+  )
+
+  const creditsEditorTextField = new ProjectEditorTextField(
+    $projectDescriptionCredits,
+    $projectDescriptionCredits.data('project-id'),
+    'credits',
```

Comment:
This should be like this for the api to work
```suggestion
    'credit',
```